src/main/java/de/hysky/skyblocker/skyblock/dwarven/PowderMiningTracker.java

```diff
@@ -0,0 +1,339 @@
+		NAME2ID_MAP.put("❈ Flawed Amethyst Gemstone", "FLAWED_AMETHYST_GEM");
+		NAME2ID_MAP.put("❈ Fine Amethyst Gemstone", "FINE_AMETHYST_GEM");
+		NAME2ID_MAP.put("❈ Flawless Amethyst Gemstone", "FLAWLESS_AMETHYST_GEM");
+
+		NAME2ID_MAP.put("☘ Rough Jade Gemstone", "ROUGH_JADE_GEM");
+		NAME2ID_MAP.put("☘ Flawed Jade Gemstone", "FLAWED_JADE_GEM");
+		NAME2ID_MAP.put("☘ Fine Jade Gemstone", "FINE_JADE_GEM");
+		NAME2ID_MAP.put("☘ Flawless Jade Gemstone", "FLAWLESS_JADE_GEM");
+
+		NAME2ID_MAP.put("⸕ Rough Amber Gemstone", "ROUGH_AMBER_GEM");
+		NAME2ID_MAP.put("⸕ Flawed Amber Gemstone", "FLAWED_AMBER_GEM");
+		NAME2ID_MAP.put("⸕ Fine Amber Gemstone", "FINE_AMBER_GEM");
+		NAME2ID_MAP.put("⸕ Flawless Amber Gemstone", "FLAWLESS_AMBER_GEM");
+
+		NAME2ID_MAP.put("✎ Rough Sapphire Gemstone", "ROUGH_SAPPHIRE_GEM");
+		NAME2ID_MAP.put("✎ Flawed Sapphire Gemstone", "FLAWED_SAPPHIRE_GEM");
+		NAME2ID_MAP.put("✎ Fine Sapphire Gemstone", "FINE_SAPPHIRE_GEM");
+		NAME2ID_MAP.put("✎ Flawless Sapphire Gemstone", "FLAWLESS_SAPPHIRE_GEM");
+
+		NAME2ID_MAP.put("✧ Rough Topaz Gemstone", "ROUGH_TOPAZ_GEM");
+		NAME2ID_MAP.put("✧ Flawed Topaz Gemstone", "FLAWED_TOPAZ_GEM");
+		NAME2ID_MAP.put("✧ Fine Topaz Gemstone", "FINE_TOPAZ_GEM");
+		NAME2ID_MAP.put("✧ Flawless Topaz Gemstone", "FLAWLESS_TOPAZ_GEM");
+
+		NAME2ID_MAP.put("❁ Rough Jasper Gemstone", "ROUGH_JASPER_GEM");
+		NAME2ID_MAP.put("❁ Flawed Jasper Gemstone", "FLAWED_JASPER_GEM");
+		NAME2ID_MAP.put("❁ Fine Jasper Gemstone", "FINE_JASPER_GEM");
+		NAME2ID_MAP.put("❁ Flawless Jasper Gemstone", "FLAWLESS_JASPER_GEM");
+
+		NAME2ID_MAP.put("Pickonimbus 2000", "PICKONIMBUS");
+		NAME2ID_MAP.put("Ascension Rope", "ASCENSION_ROPE");
+		NAME2ID_MAP.put("Wishing Compass", "WISHING_COMPASS");
+		NAME2ID_MAP.put("Gold Essence", "ESSENCE_GOLD");
+		NAME2ID_MAP.put("Diamond Essence", "ESSENCE_DIAMOND");
+		NAME2ID_MAP.put("Prehistoric Egg", "PREHISTORIC_EGG");
+		NAME2ID_MAP.put("Sludge Juice", "SLUDGE_JUICE");
+		NAME2ID_MAP.put("Oil Barrel", "OIL_BARREL");
+		NAME2ID_MAP.put("Jungle Heart", "JUNGLE_HEART");
+		NAME2ID_MAP.put("Treasurite", "TREASURITE");
+		NAME2ID_MAP.put("Yoggie", "YOGGIE");
+
+		NAME2ID_MAP.put("Goblin Egg", "GOBLIN_EGG");
+		NAME2ID_MAP.put("Green Goblin Egg", "GOBLIN_EGG_GREEN");
+		NAME2ID_MAP.put("Blue Goblin Egg", "GOBLIN_EGG_BLUE");
+		NAME2ID_MAP.put("Red Goblin Egg", "GOBLIN_EGG_RED");
+		NAME2ID_MAP.put("Yellow Goblin Egg", "GOBLIN_EGG_YELLOW");
+
+		NAME2ID_MAP.put("Control Switch", "CONTROL_SWITCH");
+		NAME2ID_MAP.put("Electron Transmitter", "ELECTRON_TRANSMITTER");
+		NAME2ID_MAP.put("FTX 3070", "FTX_3070");
+		NAME2ID_MAP.put("Synthetic Heart", "SYNTHETIC_HEART");
+		NAME2ID_MAP.put("Robotron Reflector", "ROBOTRON_REFLECTOR");
+		NAME2ID_MAP.put("Superlite Motor", "SUPERLITE_MOTOR");
+	}
+
+	@NotNull
+	private static String getItemId(String itemName) {
+		return NAME2ID_MAP.getOrDefault(itemName, "");
+	}
+
+	private static Path getRewardFilePath() {
```

Comment:
This should be a constant unless it can't be?

Reply:
It can be, but I had a better solution on another branch on another branch that is based on this so I didn't bother to change to avoid a rebase.